draft-ietf-quic-multipath.md

```diff
@@ -620,6 +620,23 @@ to abandon a path for any reason, for example, removing a hole from
 the sequence of Path IDs in use. This is not an error. The endpoint that
 receive such a PATH_ABANDON frame must treat it as specified in {{path-close}}.
 
+### Closing Duplicate Paths
+
+As noted in {{basic-design-points}}, it is possible to create paths that
+refer to the same 4-tuple. There will be cases where this is intentional,
+for example if the paths use different Differentiated Service markings.
+There may also be cases where this is not intentional, maybe as a result
+of path migration.
+
+In the non-intentional case, the endpoints
+may want to abandon one of the paths and keep the other, but
+uncoordinated Abandon from both ends of the connection may result in deleting
+two paths instead of just one. To avoid this pitfall, only the client
+SHOULD initiate the closing of duplicate paths. Applications MAY
+override this default behavior if they can coordinate path closures at part of the
+application protocol.
```

Comment:
Not sure if this normative language makes really sense to me. If you clients decided to close one of paths on the same 4-tuple, why did it open it in the first place? Or maybe the client would for some reason like to have two paths on the same 4-tuple but the server really doesn't want to have that for some other reasons. In this case I think the server will still try to close a path.

Reply:
See note above on plausible path migration.

Reply:
I removed the normative language after moving the text to the implementation section.